assets/global.js

```diff
@@ -430,13 +435,18 @@ class HeaderDrawer extends MenuDrawer {
     });
 
     summaryElement.setAttribute('aria-expanded', true);
+    window.addEventListener('resize', this.onResize);
```

Comment:
Just summarizing a conversation we had yesterday:
1) `resize` listener can be a bit heavy since it's called repeatedly when you resize a window; _**however**_
2) it's appropriate here because the overlay needs to be resized along with the browser

As you pointed out, there's a bug in production where the overlay gets cut off if the user enlarges the browser window vertically. This will fix it.